### PR TITLE
Bugfix/update naming for business

### PR DESCRIPTION
**Pull request by vertiq-jordan**
the business team requested some changes to RTD. One, to make sure that all module names have a dash in them (81-08 instead of 8108). The second was to fix the side bar such that we do not include any KVs.

### CODE REVIEW

File: docs/langs/python.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ Creating a Module
 =================
 
 Python interacts with the motor controller via Vertiq Module Objects. Currently Vertiq Python API 
-supports 3 types of motors: Vertiq 2306, Vertiq 8108, and Fortiq M42BLS
+supports 3 types of motors: Vertiq 23-06, Vertiq 81-08, and Fortiq M42BLS
```
Comment (iq-fred) [inline docs/langs/python.rst:49]: Does John have any thoughts on how we wants to hyphenate Fortiq stuff? Or does he not care about the Fortiq

File: docs/modules/vertiq_8108_150.rst [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
-.. sidebar:: Vertiq 8108 150KV
+.. sidebar:: Vertiq 81-08
 
     .. image:: ../_static/Vertiq_8108_front.jpg
-        :alt: Vertiq 8108 150KV
+        :alt: Vertiq 81-08 150KV
 
-The Vertiq 8108 150Kv is built for high performance commercial UAV and robotic applications. Its default firmware is the **Speed Firmware**.
+The Vertiq 81-08 150Kv is built for high performance commercial UAV and robotic applications. Its default firmware is the **Speed Firmware**.
```
Comment (iq-fred) [inline docs/modules/vertiq_8108_150.rst:10]: Should we lose the 150 here as well?

File: docs/modules/vertiq_8108_150.rst [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-Vertiq 8108 150KV | Speed Module
+Vertiq 81-08 | Speed Module
 ********************************
 
-.. sidebar:: Vertiq 8108 150KV
+.. sidebar:: Vertiq 81-08
 
     .. image:: ../_static/Vertiq_8108_front.jpg
-        :alt: Vertiq 8108 150KV
+        :alt: Vertiq 81-08 150KV
```
Comment (iq-fred) [inline docs/modules/vertiq_8108_150.rst:8]: Should 150 go away?

File: docs/modules/vertiq_8108_150.rst [hunk 4]
```diff
@@ -17,7 +17,7 @@ responsiveness. Vertiq motors also output advanced telemetry for motor and vehic
 
 .. note:: 
 
-    While the default firmware for the Vertiq 8108 150KV is the Speed firmware, it also fully supports the following:
+    While the default firmware for the Vertiq 81-08 150KV is the Speed firmware, it also fully supports the following:
```
Comment (iq-fred) [inline docs/modules/vertiq_8108_150.rst:20]: Same as above with the 150s

File: docs/modules/vertiq_2306_2200.rst [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
-Vertiq 2306 2200KV | Speed Module
+Vertiq 23-06 | Speed Module
 *********************************
 
-.. sidebar:: Vertiq 2306 2200KV
+.. sidebar:: Vertiq 23-06
 
     .. image:: ../_static/IQ2306_2200kv.png
-        :alt: Vertiq 2306 2200KV
+        :alt: Vertiq 23-06 2200KV
```
Comment (iq-fred) [inline docs/modules/vertiq_2306_2200.rst:8]: The 3 lines here where you changed the hyphens, should they lose the 2200 as well?

File: docs/modules/vertiq_8108_150.rst (hunk 2 above)
Comment (vertiq-jordan) [reply docs/modules/vertiq_8108_150.rst:10]: Asked Jon about this back on december 15, and never got a response. I'll poke

File: docs/langs/python.rst (hunk 1 above)
Comment (vertiq-jordan) [reply docs/langs/python.rst:49]: He is "indifferent" to how we hyphenate fortiq, and is ok with how this is now

File: docs/modules/vertiq_8108_150.rst (hunk 2 above)
Comment (vertiq-jordan) [reply docs/modules/vertiq_8108_150.rst:10]: Jon is "indifferent" to this as well. He agrees with our future plan to move towards module family pages, but is ok with this for now.

File: docs/modules/vertiq_2306_220.rst [hunk 6]
```diff
@@ -5,10 +5,10 @@ Vertiq 23-06 | Position Module
 .. sidebar:: Vertiq 23-06
 
     .. image:: ../_static/IQ2306_220kv.png
-        :alt: Vertiq 23-06 200KV
+        :alt: Vertiq 23-06 200Kv
```
Comment (iq-fred) [inline docs/modules/vertiq_2306_220.rst:8]: 200Kv? Should this be 220KV?

**Review by iq-fred — APPROVED** [review]

**Pull request merged**